types/swiper/index.d.ts

```diff
@@ -439,6 +439,7 @@ export interface SwiperOptions {
     wrapperClass?: string;
 
     // Components
+    controller?: ControllerOptions;
```

Comment:
```suggestion
    controller?: ControllerOptions | true;
```

You can update the PR by accepting the suggestion. 😉

Reply:
Updated my PR